homeassistant/components/airthings_ble/sensor.py

```diff
@@ -107,9 +115,65 @@
 }
 
 
+def migrate(hass: HomeAssistant, address: str, sensor_name: str):
```

Comment:
```suggestion
@callback
def async_migrate(hass: HomeAssistant, address: str, sensor_name: str):
```

Add `async_` to callback function to indicate they are safe to run in the event loop and do not do blocking I/O